.github/workflows/build-qa.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy QA
+on:
+  push:
+    branches:
+      - qa
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy_qa:
+    name: Deploy QA
+    runs-on: ubuntu-latest
+    environment: qa
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials for QA environment account
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::946183545209:role/GithubActionsDeployerRole
+          aws-region: us-east-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build, tag, and push image to Amazon ECR
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: scsb_item_updater
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+          docker tag $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:qa-remediation
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:qa-remediation
```

Comment:
Is this for testing and will later be updated to `qa-latest`?

Reply:
Correct. Steven recommended this approach.